bootstrap.sh

```diff
@@ -49,20 +49,20 @@ function create_standalone_venv {
 
 	# from http://stackoverflow.com/questions/4324558/whats-the-proper-way-to-install-pip-virtualenv-and-distribute-for-python
 	# Select current version of virtualenv:
-	VERSION=1.8.2
-	URL_BASE="http://pypi.python.org/packages/source/v/virtualenv"
+	VERSION=15.0.2
+	URL_BASE="https://pypi.python.org/packages/5c/79/5dae7494b9f5ed061cff9a8ab8d6e1f02db352f3facf907d9eb614fb80e9/virtualenv-15.0.2.tar.gz#md5=0ed59863994daf1292827ffdbba80a63"
```

Comment:
No probem. That direct link, with embedded hash kind of scares me. Thanks though :)